bin/sapconf

```diff
@@ -167,7 +167,7 @@ case "$1" in
       exit 1
     fi
     profile=${profile:-sapconf-netweaver}
-    echo "$profile" > /var/lib/sapconf/act_profile
+    echo "$profile" > /run/sapconf_act_profile
     tune_netweaver
```

Comment:
to be 'crash' save we need to write the 'new' profile to last_profile too, as during a 'crash' we will not have a proper 'stop' action for sapconf. Without writing to 'last_profile' here, we will lost the ability to recall after the crash the former used profile.
echo "$profile" > /var/lib/sapconf/last_profile